### ISSUE TITLE
Edge error 'Expected identifier, string or number'

### ISSUE THREAD

**Issue by SnaiCrys** [sascha245/vuex-simple#13]
[ x ] bug report
[ ] feature request
[ ] question about the decisions made in the repository
[ ] question about how to use this project

In Edge browser, using the store does not work. Error sending to:
```
/*!*************************************************************!*\
  !*** ./node_modules/vuex-simple/build/src/store-builder.js ***!
  \*************************************************************/
/*! exports provided: StoreBuilder, getStoreBuilder */
/***/ (function(module, __webpack_exports__, __webpack_require__) {

==>"use strict";
eval("__webpack_require__.r
```

index.ts 
```
import Vue from 'vue';

import VuexSimple, { getStoreBuilder } from 'vuex-simple';
import { SearchModule } from './modules/search';

Vue.use(VuexSimple);

const storeBuilder = getStoreBuilder();
storeBuilder.initialize({
    modules: {},
    strict: false,
});
storeBuilder.loadModules([SearchModule]);

const store = storeBuilder.create();

export default store;
```

**Comment by sascha245** [comment]
Hi @SnaiCrys ,

For now I couldn't reproduce the bug on my Edge browser with my samples and it seems to work just fine.

Can you give me more informations, such as the version of Edge you are using, if it works in other browsers or even better an example to reproduce the bug.

Thank you in advance,

Sascha

**Comment by SnaiCrys** [comment]
Samples also work fine for me, but if i use npm module i have a error. 

Edge 18.17763
Chrome and Firefox work fine.

The error is present even if I just import vuex-simple.

**Comment by sascha245** [comment]
Ok thanks, I managed to reproduce the error.
I will look into it and give you a response as soon as I found out something.

**Comment by sascha245** [comment]
The error should normally be fixed now in 2ea5e7b81c43e97b09abc67b54ee96d4ffc714b8 (v1.2.1)
I will let you check out and then close the issue if that did the job.

I wish you a good evening,

Sascha

**Comment by SnaiCrys** [comment]
Now everything is working properly. Thank you!
